src/lib/Canvas.ts

```diff
@@ -53,26 +52,29 @@ export interface PrintCircularOptions {
 	fit?: 'fill' | 'contain' | 'cover' | 'none';
 }
 
+export type ModuleCanvas = BrowserCanvas.Canvas | SkiaCanvas.Canvas | NodeCanvas.Canvas;
+export type ModuleImage = BrowserCanvas.Image | SkiaCanvas.Image | NodeCanvas.Image;
+
 export type GlobalCompositeOperation = CanvasRenderingContext2D['globalCompositeOperation'];
-export type AntiAlias = CanvasRenderingContext2D['antialias'];
-export type TextDrawingMode = CanvasRenderingContext2D['textDrawingMode'];
-export type PatternQuality = CanvasRenderingContext2D['patternQuality'];
+export type AntiAlias = import('canvas').CanvasRenderingContext2D['antialias'];
+export type TextDrawingMode = import('canvas').CanvasRenderingContext2D['textDrawingMode'];
+export type PatternQuality = import('canvas').CanvasRenderingContext2D['patternQuality'];
```

Comment:
these types will depend on a module that may not be present at all...

Reply:
I think it's fine (?) because those aren't standard, I have no idea what issues this might cause with `skipLibCheck` off.

Reply:
Well, in an ideal world it'll probably default to `any`, but it might also throw errors :(

Reply:
There's only one way to know, sadly, and that's by releasing `5.0.0-rc.0` under a `rc` tag.

Reply:
Canary -> Alpha -> Beta -> Release Candidate.

This would be alpha.

Reply:
`5.0.0-beta.0` then.